doc/source/conf.py

```diff
@@ -277,12 +277,17 @@ def setup(app):
     # Must be less than or equal to the XVFB window size
     pyvista.global_theme["window_size"] = np.array([1024, 768])
 
-    # suppress annoying matplotlib bug
+    # Suppress annoying matplotlib bug
     warnings.filterwarnings(
         "ignore",
         category=UserWarning,
         message="Matplotlib is currently using agg, which is a non-GUI backend, so it cannot show the figure.",
```

Comment:
```suggestion
        message="Because Matplotlib is using `agg`, which is a non-GUI backend, it cannot show the figure.",
```
The message is a run-on sentence as initially written.

Reply:
This is a warning message from Matplotlib, not from our side so we have to keep the same content to filter it.
Thanks for the rewriting though :)